gridstatus/tests/test_ercot.py

```diff
@@ -740,7 +763,7 @@ def test_get_hourly_wind_report(self):
             days=1,
         )
         df = self.iso.get_hourly_wind_report(date)
-        assert df["Publish Time"].nunique() == 1
+        assert df["Publish Time"].nunique() == 24  # One for each hour
```

Comment:
IIUC, we want to retrieve one report published every hour on the date in this situation.

Reply:
this is tricky about what would should do. 

im now remembering my original intention was to only return the latest report if no end time was provided. i feel like this logic makes sense when dealing with a reports that describe overlapping intervals

however, in the case of `ieso.get_load`, i think desired behavior is to return the full day to make it the same as other `get_load` methods.

we definitely need to fix the bug for handling "today"/`HOURLY_START`, but if just a single date is provided, im kind of indifferent on the behavior. can do the full day, or just all the data that can be fetched in one request before or equal to the supplied date

Reply:
I'll defer to you. If we want `ieso.get_load` to work correctly without affecting other "HOURLY_START` methods, then we'll need to remove the logic from the decorator and put in back in the `IESO` class.